src/app/header/menu/menu.component.html

```diff
@@ -5,20 +5,28 @@
         {{item.label}}
       </a>
       <ul class="menu-dropdown" [class.sectioned]="isMenuSectioned(item)">
-        <li *ngFor="let subitem of item.items" [class.sectioned]="subitem.__typename === 'Menu'">
+        <li *ngFor="let subitem of item.items; trackBy: trackMenuItemsFn" [class.sectioned]="subitem.__typename === 'Menu'">
           <ng-container *ngIf="subitem.__typename === 'Menu'; else normalSubMenu">
             <ng-container *ngIf="subitem.page; else noLink">
-              <a class="title" [href]="getUrl(subitem.page)" (click)="onNavigation(subitem.page, $event)">
-                {{subitem.label}}
-              </a>
+              <div class="title">
+                <a [href]="getUrl(subitem.page)" (click)="onNavigation(subitem.page, $event)">
+                  {{subitem.label}}
+                </a>
+                <button *ngIf="openedSection !== subitem.label" (click)="openSection(subitem)">&#43;</button>
+                <button *ngIf="openedSection === subitem.label" (click)="openedSection = undefined">&ndash;</button>
+              </div>
             </ng-container>
             <ng-template #noLink>
-              <span class="title">
-                {{subitem.label}}
-              </span>
+              <div class="title">
+                <span>
+                  {{subitem.label}}
+                </span>
+                <button *ngIf="openedSection !== subitem.label" (click)="openSection(subitem)">&#43;</button>
```

Comment:
Instead of &#43; you could use awesome font + icon (similarly to collapse component):
```suggestion
                <button *ngIf="openedSection !== subitem.label" (click)="openSection(subitem)"><span class="fa fa-plus"></span></button>
```

You would need to restyle the button